src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetApprenticeshipResponse.cs

```diff
@@ -14,6 +14,7 @@ public class GetApprenticeshipResponse
         public string EmployerName { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
+        public string Email { get; set; }
```

Comment:
Again do we need this

Reply:
Not yet but it is needed in the very near future

Reply:
This is fine